tables/cs-comp8.utb

```diff
@@ -34,7 +34,93 @@
 # http://teiresias.muni.cz/czbraille8/?strana=norma
 
 include digits8Dots.uti
-include cs-letterDef8Dots.uti
```

Comment:
Because cs-letterDef8Dots.uti contained `uplow` rules with two dot patterns (e.g. `uplow   \x0041\x0061  17,1`), and was used in both cs-chardefs.cti, which defines a cap sign, and cs-comp8.utb, which does not define a cap sign. For cs-chardefs.cti the `uplow` rules were replaced with `lowercase` and `base uppercase` rules, for cs-comp8.utb they were replaced with `lowercase` and `uppercase` rules.